docs/wiki/[Examples]-Deploy-Management-Resources.md

```diff
@@ -0,0 +1,50 @@
+## Overview
+This example will deploy resources for centrally managing the Enterprise Scale Landing Zone. This module will deploy the following resources with default configurations that can be adjusted once you are up and going in the subscription of your specification:
+
+
+* resource group
+* log analytics workspace
+* automation account
+* linked service to the analytics automation account
+
+
+
+The module then goes onto configure Deploy if Not Exist policies to enable features within Log Analytics and Sentinel including
+but not limited to:
+* enabling monitoring for arc
+* enabling monitoring for vms
+* enabling monitoring for vmss
+* enabling defender for dns
+* much more...
+
+
+
+> PREREQUISITE: Ensure the module version is set version  0.2.0 or greater
+```hcl
+     module "enterprise_scale" {
+      source = "Azure/caf-enterprise-scale/azurerm"
+      version = "0.3.1"
+     }
+```
+
+If upgrading to a later version of this module, make sure to run `terraform init`
+
+To initiate the resource creation,  `deploy_managagement_resources` must be set to true. The  `subscription_id_management` is a required variable so that the resources have somewhere to land. Enabling these two variables will deploy ~200 resources mostly consisting of Deploy if Not Exist policies and RBAC permissions.
```

Comment:
`subscription_id_management` is actually only used within the data model to ensure the correct Subscription ID is used in things like resource IDs and when setting values for use in Policy Assignments, etc.

Resource deployment happens in the scope of the current `provider` configuration (i.e. the current context of the user credentials used to deploy resources with Terraform) so both conditions must be true and equal for a successful deployment.

You already start to touch on this below, but in our accelerator repository, you can see how we are using aliases to create an example of how to manage this in a slightly more complex (level 300+) deployment.

- Example [providers.tf](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale-accelerator/blob/staging/src/modules/enterprise-scale/providers.tf) file
- Example [main.tf](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale-accelerator/blob/staging/src/modules/enterprise-scale/main.tf) file

In truth, this is really a requirement to make the module work properly and to its full potential.

Reply:
Updated. Leaving item open for review